website/sidebars.json

```diff
@@ -15,22 +15,20 @@
     ],
     "ORY Hydra": [
       "hydra/index",
-      "hydra/install",
+      "hydra/oauth2",
```

Comment:
Or alternatively have index and oauth2 as root things and then one section "Integrating Hydra in your Stack" with implementing consent and login-cosnent flow?

Reply:
I went with your first suggestion and have created a "Concepts" section. I omit "theory" since that sounds too academic for what we are doing here (engineering a pragmatic solution of an existing standard).